assets/src/edit-story/components/checklist/videoOptimizationCheckbox/index.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { TranslateWithMarkup, __ } from '@web-stories-wp/i18n';
+import styled from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import { trackClick } from '../../../../../../packages/tracking/src';
+import {
+  Checkbox,
+  Link,
+  Text,
+  THEME_CONSTANTS,
+} from '../../../../design-system';
+import { useConfig, useCurrentUser } from '../../../app';
+import { CARD_WIDTH } from '../../helpCenter/navigator/constants';
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: ${CARD_WIDTH}px;
+  margin: 16px 0;
+  padding: 16px;
+  background-color: ${({ theme }) => theme.colors.bg.secondary};
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+`;
+
+const CheckboxContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+`;
+
+const DescriptionText = styled(Text).attrs({
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL,
+})`
+  color: ${({ theme }) => theme.colors.fg.secondary};
+  margin-bottom: 16px;
+`;
+
+const CheckboxLabel = styled(DescriptionText)`
+  margin-left: 8px;
+  margin-bottom: 0;
+  line-height: 20px;
+`;
+
+function VideoOptimizationCheckbox() {
+  const { dashboardSettingsLink } = useConfig();
```

Comment:
Let's also make sure that the user has the ability to upload files before showing this. 
```
  const {
    capabilities: { canUploadFiles } = {},
  } = useConfig();
```

So then this check would show up if web_stories_media_optimization is false _and_ canUploadFiles is true